include/http/router.js

```diff
@@ -55,62 +56,62 @@ module.exports = function (pb) {
         }
 
         static replaceMiddleware(name, middleware) {
-            let index = this._indexOfMiddleware(name);
-            if(index >= 0 && middleware) {
-                this.internalMiddlewareStack[index] = middleware;
+                let index = this._indexOfMiddleware(name);
+                if (index >= 0 && middleware) {
+                    this.internalMiddlewareStack[index] = middleware;
+                }
             }
-        }
-        /*****
-         * Internal Helper Functions
-         */
-        static get internalMiddlewareStack () {
-            if(this._internalMiddlewareStack)
+            /*****
+             * Internal Helper Functions
+             */
+        static get internalMiddlewareStack() {
+            if (this._internalMiddlewareStack)
```

Comment:
We should probably add braces to all if statements.  I am a bad person and forgot them it seems...

Reply:
Done :P